LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWAbilitiesModTemplate.uc

```diff
@@ -56,6 +56,13 @@ static function UpdateAbilities(X2AbilityTemplate Template, int Difficulty)
 			// reintroduce it.
 			// UpdateShadow(Template);
 			break;
+		case 'Bayonet':
+		UpdateBayonet(Template);
```

Comment:
Would you mind making the indent consistent, please? As in the code in a `case` should be indented from it.